app/src/main/java/com/example/bigowlapp/activity/FingerprintAuthenticationActivity.java

```diff
@@ -127,6 +128,9 @@ protected void onStart() {
         if (homePageViewModel == null) {
             homePageViewModel = new ViewModelProvider(this).get(HomePageViewModel.class);
         }
+        if (phoneNumberFormatter == null) {
+            phoneNumberFormatter = new PhoneNumberFormatter(this);;
```

Comment:
Just like what @refatK stated.

This semi-colon should be removed; it's also a code smell according to sonarqube.